### ISSUE TITLE
dApps - Can't sign a message from Uniswap

### ISSUE THREAD

**Issue by anastasiyaig** [status-im/status-desktop#15946]
# Bug Report

## Description

Signing message from uniswap gets me to rejected state

## Steps to reproduce

https://github.com/user-attachments/assets/715f173e-3b22-42c4-8d1a-c2369d04886f

#### Actual behavior

```
ERR 2024-08-01 07:14:49.835+03:00 rpc response error                         topics="rpc" tid=27431383 file=core.nim:36 err="\nstatus-go error [methodName:wallet_safeSignTypedDataForDApps, code:-32000, message:unknown type \"uint160\" ]\n"
ERR 2024-08-01 07:14:49.835+03:00 error doing rpc request                    topics="rpc" tid=27431383 file=core.nim:39 methodName=wallet_safeSignTypedDataForDApps exception="\nstatus-go error [methodName:wallet_safeSignTypedDataForDApps, code:-32000, message:unknown type \"uint160\" ]\n"
DBG 2024-08-01 07:14:49.835+03:00 qt message 
```

### Additional Information

- Status desktop version: 2.30 RC4
- Operating System: MacOS

**Comment by clauxx** [comment]
Seems like this issue is caused by the old version of geth on status-go, which doesn't include all solidity types for typed data, which are supposed to be included based on EIP-712. Hence, requests that include e.g. `uint160`, `uint48`, etc. in the types are considered invalid for now. The issue seems fixed on [go-ethereum master](https://github.com/ethereum/go-ethereum/blob/de6d5976794a9ed3b626d4eba57bf7f0806fb970/signer/core/apitypes/types.go#L847), where they support all types necessary. 

Can be moved to 2.31 for now, as we definitely cannot fix it in 2.30.

**Comment by alexjba** [comment]
Closed by https://github.com/status-im/status-go/pull/5755
